ts/components/ContextualHelpModal.tsx

```diff
@@ -153,54 +164,34 @@ const ContextualHelpModal: React.FunctionComponent<Props> = (props: Props) => {
       onRequestClose={onClose}
     >
       <Container>
-        <BaseHeader
-          accessibilityEvents={{
-            avoidNavigationEventsUsage: true
-          }}
-          headerTitle={I18n.t("contextualHelp.title")}
-          customRightIcon={{
-            iconName: "io-close",
-            onPress: onClose,
-            accessibilityLabel: I18n.t(
-              "global.accessibility.contextualHelp.close"
-            )
-          }}
-        />
-
-        {!contextualHelpData.content && (
-          <View centerJustified={true}>
-            <ActivityIndicator color={themeVariables.brandPrimaryLight} />
-          </View>
-        )}
-        {contextualHelpData.content && (
-          <Content
-            contentContainerStyle={styles.contentContainerStyle}
-            noPadded={true}
-          >
-            <H3 accessible={true}>{contextualHelpData.title}</H3>
-            <View spacer={true} />
-            {contextualHelpData.content}
-            <View spacer={true} />
-            {contextualHelpData.faqs && isContentLoaded && (
-              <FAQComponent
-                onLinkClicked={props.onLinkClicked}
-                faqs={contextualHelpData.faqs}
-              />
-            )}
-            {isContentLoaded && (
-              <React.Fragment>
-                <View spacer={true} extralarge={true} />
-                <InstabugAssistanceComponent
-                  requestAssistance={reportType =>
-                    props.onRequestAssistance(reportType, props.supportToken)
-                  }
-                />
-              </React.Fragment>
-            )}
-            {isContentLoaded && <EdgeBorderComponent />}
-          </Content>
+        {showSendPersonalInfo ? (
+          <SendSupportTokenInfo
+            onClose={onClose}
+            onGoBack={() => setShowSendPersonalInfo(false)}
+            requestAssistance={(
+              reportType: BugReporting.reportType,
+              sendToken: boolean
+            ) =>
+              requestAssistance(
+                reportType,
+                sendToken,
+                props.onRequestAssistance,
+                props.supportToken
+              )
+            }
+          />
+        ) : (
+          <ContextualHelpHome
```

Comment:
```suggestion
          <ContextualHelpComponent
```
Could it be a more descriptive name?